### ISSUE TITLE
Errors in execution with draw.textsize

### ISSUE THREAD

**Issue by mrinnella60** [mstelz/vas_card_generator#1]
OK. I've overcome a couple of problems, but still running into the following error:

DevTools listening on ws://127.0.0.1:62752/devtools/browser/f47bcbdc-16d2-4c0f-b96a-d2b3e8b59789

Saving screenshot to: output_images\Essex.png

Traceback (most recent call last):

File "C:\Users\mrinn\Downloads\vas_card_generator-main\generate2.py", line 211, in <module>draw_vertical_lines(screenshot_path, int(str(ship["hull"]).split("/")[0][0]))

~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

File "C:\Users\mrinn\Downloads\vas_card_generator-main\generate2.py", line 97, in draw_vertical_lines

text_width, text_height = draw.textsize(text, font=font)

^^^^^^^^^^^^^

AttributeError: 'ImageDraw' object has no attribute 'textsize'

I'm hoping you can help me out with this as I am not a python developer.

**Comment by mstelz** [comment]
You are right, I came across this last night myself. This is a discrepancy between pillow versions. I'll get it sorted but in the mean time I actually pushed up an executable for Linux and Windows last night. In the meantime check out the Readme "Running via exectuable" section for using those as those have the proper versions and bundled together.

**Comment by mrinnella60** [comment]
Unfortunately, this happened when I tried to download:
![Screenshot (1)](https://github.com/user-attachments/assets/415ae736-e055-4677-a20b-f0c79e8be4a6)
Essentially, a Trojan was found, specifically Trojan.Script/Wacatac.B!ml

**Comment by mstelz** [comment]
Well that's odd, I just created it using pyinstaller. My system did not find anything. Otherwise in the next day or so I can update the script to work with the newer pillow versions

**Comment by mstelz** [comment]
https://github.com/mstelz/vas_card_generator/actions/runs/12075799822/job/33676174637

I scanned the repo as well using clamAV it provided no results

**Comment by mstelz** [comment]
@mrinnella60 try pulling the latest codebase down, it should work with the newer APIs from pillow, otherwise the EXE is fine, scanned it with Windows Defender, MalwareBytes and ClamAV

**Comment by mrinnella60** [comment]
I don't know why WIndows Defender is being so difficult with me on the executable.  In any case, I updated the latest codebase and it is working using the python generate.py command so I'm happy.  I did download the executable separately and renamed the resulting "unconfirmed.crdownload" file to vascardgenerator.exe and that worked as well.  I certainly appreciate the quick response on this and will be happily creating cards for all my ships and using your STLs as well or possible designing my own.  I'm going to see what I can find for ship images and if it's alright with you I will upload them to your ship_images collection.
